.travis.yml

```diff
@@ -26,8 +26,9 @@ before_script:
   - touch config/settings.local.yml
   - bundle exec rake db:migrate
   - bundle exec rake i18n:js:export
+  - bundle exec rake yarn:install
 script:
   - bundle exec rubocop -f fuubar
-  - bundle exec rake eslint:run_all
+  - bundle exec rake eslint
```

Comment:
We should add this to CONTRIBUTING.md in the Coding Style section. I mean, I expect a change like this to have a corresponding change to the documentation, but the documentation mentioning eslint is missing :smile: